src/test/resources/failing-fetcher-schema.txt

```diff
@@ -15,13 +15,6 @@
 # * specific language governing permissions and limitations
 # * under the License.
 
-# This directive maps fields to our Sling data fetchers
```

Comment:
We should leave tests where these directives are defined in the schema as before, to make sure these old schemas continue to work. I think doing that for the @fetcher and @resolver directives is sufficient as we didn't release this module with the @connection implemented.